packages/tsconfig/base.json

```diff
@@ -24,7 +23,7 @@
     "noUnusedParameters": true,
     "resolveJsonModule": true,
     "target": "ESNext",
-    "module": "ES2022"
+    "module": "preserve"
```

Comment:
what are the implications of this new module config?

Reply:
there are zero implications because no packages are published from this repo. but due to several idiosyncratic configurations and webpack's lack of true support for typescript config, the previous module resolution prevented the import of the package json data in webpack config, and this resolution mode is more permissive.